tests/cursor/bug1839-001.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+PHPC-1839: Referenced, out-of-scope, non-interned string in typeMap
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
```

Comment:
Just curious, but do any of these tests actually need a live server? Could you test the same code path using [`MongoDB\BSON\toPHP()`](https://www.php.net/manual/en/function.mongodb.bson-tophp.php) and a simple BSON document (e.g. `0x05` followed by four null bytes).